hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/job/impl/TestMapReduceChildJVM.java

```diff
@@ -58,20 +58,20 @@ public void testCommandLine() throws Exception {
     app.verifyCompleted();
 
     Assert.assertEquals(
-      "[" + MRApps.crossPlatformify("JAVA_HOME") + "/bin/java" +
-      " -Djava.net.preferIPv4Stack=true" +
-      " -Dhadoop.metrics.log.level=WARN " +
-      "  -Xmx820m -Djava.io.tmpdir=" + MRApps.crossPlatformify("PWD") + "/tmp" +
-      " -Dlog4j.configuration=container-log4j.properties" +
-      " -Dyarn.app.container.log.dir=<LOG_DIR>" +
-      " -Dyarn.app.container.log.filesize=0" +
-      " -Dhadoop.root.logger=INFO,CLA -Dhadoop.root.logfile=syslog" +
-      " org.apache.hadoop.mapred.YarnChild 127.0.0.1" +
-      " 54321" +
-      " attempt_0_0000_m_000000_0" +
-      " 0" +
-      " 1><LOG_DIR>/stdout" +
-      " 2><LOG_DIR>/stderr ]", app.launchCmdList.get(0));
+        "[" + MRApps.crossPlatformify("JAVA_HOME") + "/bin/java" +
```

Comment:
avoid space

Reply:
I didn't want to change this. But check style problem show 
in https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6102/5/artifact/out/results-checkstyle-root.txt. 
```
./hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/job/impl/TestMapReduceChildJVM.java:67:      " -Dyarn.app.container.log.filesize=10240" +: '" -Dyarn.app.container.log.filesize=10240"' has incorrect indentation level 6, expected level should be 8. [Indentation]
./hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/test/java/org/apache/hadoop/mapreduce/v2/app/job/impl/TestMapReduceChildJVM.java:172:      " -Dyarn.app.container.log.filesize=10240" +: '" -Dyarn.app.container.log.filesize=10240"' has incorrect indentation level 6, expected level should be 8. [Indentation]
```